src/main/java/io/pinecone/unsigned_indices_model/QueryResponseWithUnsignedIndices.java

```diff
@@ -8,13 +8,43 @@
 import java.util.Collections;
 import java.util.List;
 
+/**
+ * This class represents the response to a query, where the `ScoredVector` objects in the `matches` list
+ * contain `SparseValuesWithUnsignedIndices` instead of the standard `SparseValues`.
+ * <p>
+ * The `QueryResponseWithUnsignedIndices` class contains the following fields:
```

Comment:
I'm not sure if the backticks work for the documentation here. I tried it using the `gradle generateJavadoc` command and it didn't seem to apply any styling.

You can use the `{@link QueryResponseWithUnsignedIndices}` syntax for things that are definitions of classes to provide actual documentation links.